Makefile

```diff
@@ -91,9 +91,6 @@ centos-% fedora-% rhel-%: checkout ## build rpm packages for the specified distr
 debian-% raspbian-% ubuntu-%: checkout ## build deb packages for the specified distro
 	$(MAKE) -C deb VERSION=$(VERSION) GO_VERSION=$(GO_VERSION) $@
 
-.PHONY: static
-static: DOCKER_BUILD_PKGS:=static-linux cross-mac cross-win cross-arm
-static: checkout ## build static-compiled packages
-	for p in $(DOCKER_BUILD_PKGS); do \
-		$(MAKE) -C $@ VERSION=$(VERSION) GO_VERSION=$(GO_VERSION) TARGETPLATFORM=$(TARGETPLATFORM) CONTAINERD_VERSION=$(CONTAINERD_VERSION) RUNC_VERSION=$(RUNC_VERSION) $${p}; \
-	done
+.PHONY: static-%
+static-%: checkout ## build static-compiled packages for the specified os-arch
+	$(MAKE) -C static VERSION=$(VERSION) GO_VERSION=$(GO_VERSION) $@
```

Comment:
note to self: we need to get rid of passing (at least) `GO_VERSION`, so that we can pick the "default" if things are not set.